server/test-data/editor-test.ts

```diff
@@ -0,0 +1,21 @@
+/** Section Definition */
+export interface SectionTest {
+  /** Name of section */
+  readonly name: string;
+}
+
+/** Author Definition */
+export interface AuthorTest {
+  /** Name of Author */
+  readonly name: string;
+}
+
+export interface StoryTest {
+  /** Name of story */
+  readonly name: string;
+  /** List of sections story belongs to */
+  readonly sections: ReadonlyArray<SectionTest>;
+
+  /** List of story authors */
+  readonly authors: ReadonlyArray<AuthorTest>;
+}
```

Comment:
I think this is already defined here https://github.com/quintype/quintype-node-migration-helpers/blob/master/src/lib/editor-types.ts#L124

Reply:
No it is defined in different folder (From client-migration)